ci/environment.yml

```diff
@@ -7,6 +7,7 @@ dependencies:
   - esmpy>=8.2.0
   - mpich
   - netcdf4
+  - numba
```

Comment:
I'm curious why `numba` is required here.

Reply:
this issue might be relevant to this https://github.com/pangeo-data/xESMF/issues/158

Reply:
removed it and everything seems to work fine...